react/src/js/react/posts/Post.jsx

```diff
@@ -215,14 +221,8 @@ class Post extends Reflux.Component {
             );
         }
 
-        const post = this.props.post;
-        let posts = [post];
-        if (this.props.images) {
-            posts = posts.concat(this.props.images);
-        }
-        if (post) console.log(post);
-        if (post && post.author) console.log(post.author);
-        if (post && post.author && post.author.url) console.log(post.author.url);
+        const post = this.props.post,
+            images = post.images;
```

Comment:
What if the post doesn't have an images attribute (i.e. posts from other servers?)

Reply:
This should still handle it in the render content, since the other groups have their posts as either images or posts......